### ISSUE TITLE
Allow a User/Contributor to post a notice via an API endpoint

### ISSUE THREAD

**Issue by lutangar** [dis-moi/backend#333]

**Comment by MaartenLMEM** [comment]
@Goutte what's the size ?

**Comment by MaartenLMEM** [comment]
Salut @Tonio @lutangar , questions sur le calendrier de l'API v4.
Au moins de 2 projets sont très dépendants de la nouvelle API :
L'automation des contributions 
L'interface de contributions avancée
J'aurais besoin des quelques éclairages : 
- Est-ce que les 2 projets ont le même délai d'attente de mis en prod de l'API ou bien, un version simplifiée peut déjà convenir à l'automatisation de contributions? 
- Pour chacun de ces 2 projets, quelle prévision de mise à disposition de l'API peut on faire ?

**Comment by MaartenLMEM** [comment]
Salut @Goutte  @lutangar  , questions sur le calendrier de l'API v4.
Au moins de 2 projets sont très dépendants de la nouvelle API :
L'automation des contributions 
L'interface de contributions avancée
J'aurais besoin des quelques éclairages : 
- Est-ce que les 2 projets ont le même délai d'attente de mis en prod de l'API ou bien, un version simplifiée peut déjà convenir à l'automatisation de contributions? 
- Pour chacun de ces 2 projets, quelle prévision de mise à disposition de l'API peut on faire ?

**Comment by lutangar** [comment]
@MaartenLMEM this issue was about posting from the extension and this is already in production.
I'm closing this and re-posting your comment in the new issue.

**Comment by lutangar** [comment]
My bad this is the correct issue, I thought there was one with a description but nop :/

**Comment by lutangar** [comment]
@Goutte correct me if I'm wrong but I think you said it was possible to ship "posting a notice" before the authentication part. But maybe the authentication part is not far from release?

**Comment by Goutte** [comment]
Yes !  I'm almost there (I hope).  We still need to test and figure out which properties of a Notice are writable.  I'm working on it right now if you want a call (I'd like one).
